lib/src/client_options.dart

```diff
@@ -134,6 +134,9 @@ class GatewayIntents {
   /// Includes events: `TYPING_START`
   static const int directMessageTyping = 1 << 14;
 
+  /// Includes events: `GUILD_SCHEDULED_EVENT_CREATE`, `GUILD_SCHEDULED_EVENT_DELETE`, `GUILD_SCHEDULED_EVENT_UPDATE`, `GUILD_SCHEDULED_EVENT_USER_ADD`, `GUILD_SCHEDULED_EVENT_USER_REMOVE`
```

Comment:
I'm not adding last to events beacuse they are experimental and I don't want to deal with it later. Will add if they become stable